src/actions/printers/reading.js

```diff
@@ -98,6 +106,12 @@ export const getWebcamSnapshot = createHttpAction(
         if (webcams.queue && webcams.queue[id]) {
           const timeoutData = webcams.queue[id];
 
+          // no matter what, we always kill ani potentional timeout if set before setting a new one
+          // this prevents ghost intervals staying in the background
```

Comment:
Let me be your spellcheck :)

```suggestion
          // kill any potential timeout before setting a new one
          // this prevents ghost intervals staying in the background
```